### DISCUSSION TITLE
How to prepare Android device or Emulator for epbf programs

### DISCUSSION THREAD

**Discussion by kochia3** [gojue/ecapture#194] [Q&A] [answered]
After we have compiled epbf program build with libbpfgo or similar toolset, how to prepare an Android device or an Emulator to run them?
I guess Emulator is easy way but still I failed to run ebpf programs

**Comment by cfc4n** [comment]
same as on linux, for example: `./ecapture tls`, I will update a video about use ecapture on android next week.

**Comment by cfc4n** [comment, marked as answer]
@kochia3 if you know Chinese , you can read https://mp.weixin.qq.com/s/KWm5d0uuzOzReRtr9PmuWQ for how to run eCapture on android.   or I will post English version next week.
